### DISCUSSION TITLE
"setup.py test" before contribution of a new feature

### DISCUSSION THREAD

**Discussion by Omri-L** [pyg-team/pytorch_geometric#2291] [Q&A] [answered]
Hi,
I've followed the contribution readme file and I have a question (sorry if it is a silly one :) ) -
when I'm running the setup.py test script - what are the meanings of "stmts" and "miss" values? 
If I've added new files and the cover value is low what should I do?

Thanks,
Omri.

**Comment by rusty1s** [comment, marked as answer]
`Stmts` is the number of total statements in a specific file. For example, for the first line, 26 statements were executed, out of which 8 were missed, so `18/26=0.69` were covered. In general, you do not need to worry about that too much. We can always add tests later on.

**Reply by Omri-L** [reply]
Great! So I am adding now a pull request of MeshCNN (this is done with a full cooperation of the author).
